include/ceed.h

```diff
@@ -143,6 +143,10 @@ CEED_EXTERN CeedRequest *const CEED_REQUEST_IMMEDIATE;
 CEED_EXTERN CeedRequest *const CEED_REQUEST_ORDERED;
 CEED_EXTERN int CeedRequestWait(CeedRequest *req);
 
+CEED_EXTERN CeedElemRestriction CEED_RESTRICTION_IDENTITY;
+CEED_EXTERN CeedBasis CEED_BASIS_COLOCATED;
+CEED_EXTERN CeedVector CEED_QDATA_NONE;
```

Comment:
I think we need a better name because "qdata" is no longer a public concept (I think "none" doesn't quite fit the old meaning either). Perhaps `CEED_VECTOR_ACTIVE` to indicate that this vector will be supplied as an active input or output? The use for weights might need a different name.

Also, we need Doxygen comments for these new values.

Reply:
Updated